opentech/apply/projects/models.py

```diff
@@ -6,6 +6,11 @@ class Project(models.Model):
 
     name = models.TextField()
 
+    contact_legal_name = models.TextField(default='')
+    contact_email = models.TextField(default='')
+    contact_address = models.TextField(default='')
+    value = models.TextField(default='')
```

Comment:
Value is an integer in the submission stage, but we'll probably want to change that. We should go decimal here because money.

Reply:
Fixed and rebased to avoid extra migrations.